lib/active_merchant/billing/gateways/quickbooks.rb

```diff
@@ -0,0 +1,312 @@
+begin
+  require "oauth"
+rescue LoadError
+  raise "OAuth is required to use QuickBooks Payments. Please run `gem install oauth`."
```

Comment:
I'm actually against the prescriptive "Please run `gem install oauth`.", since in many cases one would be actually adding to their Gemfile. Maybe "You need to have the oauth gem installed in order to use the ActiveMerchant QuickbooksGateway adapter."?